tests/test_strategy.py

```
@@ -528,6 +582,175 @@ def test_classic_bandit_mixed_actions(
+    ----------
+    score : float
+        Score to verify.
+    """
+    d = DynamicPricingBandit()
+    assert d._verify_action(score) is True
+
+
+@patch("pybandits.strategy.maximize_by_quantity")
+def test_dynamic_pricing_quantitative_action(
+    mock_maximize: MagicMock,
+    return_value: np.ndarray = np.array([0.3, 0.7]),
```

Ruff (0.15.15)

[warning] Do not perform function call `np.array` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)